### ISSUE TITLE
The Datepicker is not accessible completely when the Device font size is increased on IOS using the device Text Accessibility Options

### ISSUE THREAD

**Issue by sagarmph** [henninghall/react-native-date-picker#634]
The Datepicker is not accessible completely when the Device font size is increased on IOS using the device Text Accessibility Options
The button **Confirm** is overlapping on the Datepicker component

<img width="327" alt="Screenshot 2023-04-10 at 1 05 35 PM" src="https://user-images.githubusercontent.com/121793503/230851820-65d6f257-47ce-43db-9c47-de3d91208b9c.png">

**Expected behavior**
In the modal version of Datepicker, All the elements needs to be properly accessible, when device Text Accessibility Options are changed

**Smartphone (please complete the following information):**
 - OS: [Android or iOS]
 - React Native version [e.g. 0.59.4]
 - react-native-date-picker version [e.g. 2.5.1]

**Comment by nguyenminhkhmt** [comment]
I got the same issue, any idea?

**Comment by devontivona** [comment]
+1

**Comment by panchal-krunal** [comment]
Any solution on this ?

**Comment by qaezrun** [comment]
> Any solution on this ? (1)

**Comment by AndrewDeminWeb** [comment]
+1

**Comment by anil-punchh** [comment]
Hey @henninghall, I'm still getting this issue with the font scaling(the Device font size is increased on IOS).


OS: iOS
"react-native": "0.79.5"
"react-native-date-picker": "5.0.13"

<br>

<img width="413" height="838" alt="Image" src="https://github.com/user-attachments/assets/43fc6340-ab6c-4fc0-a7e3-047cb80b2da7" />
